### PR TITLE
Review for project from trials (tribute page)

### PR DESCRIPTION
**Pull request by TamaraDimov**

### CODE REVIEW

**Review by TamaraDimov — COMMENTED** [review]
Hi @TamaraDimov ,

Your project page is looking fantastic so far 🤩 !
There are some issues that you still need to work on to go to the next project but you are almost there!

image

Required Changes ♻️

- > Add descriptive README.md file. 

- > Good job creating initializing startup files 👍 , but you need to install the linters and check your code. Kindly install the linters by following the steps in this [documentation.](https://github.com/microverseinc/linters-config/tree/master/html-css#webhint). You will understand more about linters by reading carefully. 💯


- > Unfortunately you forget to create .gitignore. Kindly create it and make sure that it contains node_modules. Please read more about the use of the .gitgnore if needed 👍

File: index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        />
+        <div id="img-caption">
+        <p id="tribute-info">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </p>
+    </div>
+      </div>
+    </header>
+      <div class="main-text">
```
Comment (TamaraDimov) [inline index.html:30]: - [ ] Kindly use `<section>` for this. This is semantically correct and screen readers are able to read your code easily. Each individual area of your project should be a section on its own

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        If you have time, you should read more about this incredible human being
+        on his
+        <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank"
+          >Wikipedia entry.</a
+        >
+      </h5>
+
+      <p><em>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."
+
+        -- Indian Prime Minister Manmohan Singh</em></p>
```
Comment (TamaraDimov) [inline index.html:94]: - [ ] Please group this part of your code. It should be easy for the reader of the code to understand what is this text for and easier for you to style it.

**Comment by andgarzonmal** [comment]
Hi @TamaraDimov, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good implementations  ✔️ 
- Nice job doing the review ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**